cognite/neat/graph/transformers/_prune_graph.py

```diff
@@ -0,0 +1,115 @@
+from rdflib import RDF, Graph, Namespace, URIRef
+from rdflib.term import Node
+
+from ._base import BaseTransformer
+
+
+class ResolveValues(BaseTransformer):
+    """
+    Knowledge graph pruner and resolver. Will remove rdf triples from graph that does not have connections
```

Comment:
I think you need to a bit update this docs as you are describing both pruner and resolver.